Deployment/README.md

```diff
@@ -0,0 +1,105 @@
+# CLI Build Tools
+
+## Prereqs
+
+This project uses .NET Framework 4.7.
+
+The build process uses [psake](https://github.com/psake/psake).
+It has been included as a submodule, before running any build commands run `git submodule update --init --recursive`.
+
+psake 4.7.4 uses the [Visual Studio Setup PowerShell Module](https://github.com/Microsoft/vssetup.powershell) to detect the msbuild executable.
+You can read more about [their interactions here](https://github.com/psake/psake/issues/241).
+
+## Running Tests
+
+This project uses NUnit 3.
+All test projects included in the Solution should be picked up during a Unit Test Build.
+Currently, Integration Tests must be run manually in Visual Studio.
+
+In order to rebuild and run with unit tests pass `UnitTest` as the -TaskList parameter to the build script.
+The following are all valid invocations:
+
+```ps1
+.\build.ps1 # the default is a unit test build
+.\build.ps1 UnitTest
+.\build.ps1 -TaskList UnitTest
+```
+
+Note: The -Version parameter is ignored for UnitTest builds.
+
+## Creating a RAP file
+
+This project uses the [RelativityDev.RapBuilder nuget package](https://www.nuget.org/packages/RelativityDev.RapBuilder/0.0.0.3-alpha) to build a rap file.
+
+In order to rebuild and package the application as a rap pass `CreateRap` as the -TaskList parameter to the build script.
+Additionally, you can pass a version for the rap to the -Version parameter. 
+The following are all valid invocations:
+
+```ps1
+.\build.ps1 CreateRap # the version defaults to 1.0.0.18
+.\build.ps1 CreateRap 9.6.0.0
+.\build.ps1 CreateRap -Version 9.6.0.0
+.\build.ps1 -TaskList CreateRap 
+.\build.ps1 -TaskList CreateRap 9.6.0.0
+.\build.ps1 -TaskList CreateRap -Version 9.6.0.0
+```
+
+You can find the output rap in the Deployment/Packages folder.
+*Note:* the output rap will override any other raps with the same version.
+
+## Windows Subsystem for Linux
```

Comment:
I don't think we should include WSL stuff. It adds confusion for the other 99% of people and can lead people to think this builds cross-platform.